google/cloud/logging_v2/logger.py

```diff
@@ -200,18 +201,28 @@ def log_text(self, text, *, client=None, **kw):
         self._do_log(client, TextEntry, text, **kw)
 
     def log_struct(self, info, *, client=None, **kw):
-        """Log a dictionary message
+        """Logs a dictionary message.
 
         See
         https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/write
 
+        The dictionary entry must be able to be serializable to a Protobuf Struct
+        (see https://protobuf.dev/reference/protobuf/google.protobuf/#value for more
+        details), otherwise it will not be logged.
```

Comment:
"Otherwise a ValueError exception will be raised?"

Also, maybe include that this essentially means strings, floats, bools, lists, and dicts, and None?